src/vivarium_census_prl_synth_pop/tools/make_results.py

```diff
@@ -544,6 +541,9 @@ def subset_results_by_state(processed_results_dir: str, state: str) -> None:
             output_file_path = state_observer_dir / usa_obs_file.name
             if output_file_path.exists():
                 continue
-            state_data = read_datafile(usa_obs_file, reset_index=False, state=state)
+            if observer == metadata.DatasetNames.SSA:
```

Comment:
@rmudambi I went back and forth on which is better - this implementation or setting state=None or the state provided in the cli command and then having one call to read_datafile. I think the method I implemented is easier to understand but not as clean.

Reply:
I think I prefer setting `state = None`, since you remove the duplication of calls to `read_datafile`, but you need to be careful that you do this by defining a new variable, since we don't want to change the existing value of `state` since that will be used by other datasets in subsequent iterations of the loop.